guides/common/modules/proc_setting-destruction-of-the-virtual-machine-upon-host-deletion.adoc

```diff
@@ -0,0 +1,36 @@
+[id="setting-destruction-of-the-virtual-machine-upon-host-deletion_{context}"]
+= Setting destruction of the virtual machine upon host deletion
```

Comment:
IMHO the heading you had before was simpler for non-English speakers, but if you want to go with this, I suggest something like:

```suggestion
= Destroying the virtual machine upon host deletion
```

Reply:
I still think "destroying" is not the right verb. I did a Google search and "delete" or "remove" are used far more often than "destroy". VMware uses "remove", IBM uses "delete". I did not find "destroy" used by any recognized vendors.

Reply:
Okay. Then I might go with "Removing the virtual machine upon host deletion". Sounds good?